app/views/admin/admins/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="container">
+  <div class="row">
+    <div id="side-menu"  class="col-sm-2">
+      <ul id="links-list">
+        <% if user_signed_in? && current_user.admin == true %>
```

Comment:
no need for == true here...the value itself is true or false.

Reply:
this time around you will be sure that the current user is present as you have validations in controller now..so no need to check if signed in.